scanpy/preprocessing/_pca.py

```diff
@@ -73,6 +74,11 @@ def pca(
         Efficient computation of the principal components of a sparse matrix
         currently only works with the `'arpack`' or `'lobpcg'` solvers.
 
+        If X is a dask array, then `'randomized'` will use the randomized algorithm
+        from dask-ml. Otherwise, `'auto'` will be given to `dask_ml.decomposition.PCA` and
+        `'tsqr'` will be given to `dask_ml.decomposition.TruncatedSVD`.
```

Comment:
```suggestion
        from dask-ml. Otherwise, `'auto'` will be given to :class:`~dask_ml.decomposition.PCA` and
        `'tsqr'` will be given to :class:`~dask_ml.decomposition.TruncatedSVD`.
```